assets/quill-cursors.scss

```diff
@@ -91,26 +97,22 @@ $transition-func: cubic-bezier(0.250, 0.460, 0.450, 0.940);
     }
   }
 
-  .ql-cursor-flag:hover,
-  .ql-cursor-flag.show-flag,
-  .ql-cursor-caret-container:hover + .ql-cursor-flag {
-    opacity: 1;
-    visibility: visible;
-    transition: none;
-  }
-
   .ql-cursor-flag.no-delay[style] {
```

Comment:
We should probably move the delay logic into JS as well — I'm not sure this really works any more?